demo/src/configs/arsaboo/lovelace.ts

```diff
@@ -46,7 +46,7 @@ export const demoLovelaceArsaboo: () => LovelaceConfig = () => ({
   title: "ARS Home",
   views: [
     {
-      icon: "mdi:home-assistant",
+      icon: "hademo:home-assistant",
```

Comment:
We shouldn't use the set in Lovelace, and `home-assistant` is in the `hass:` set, as is `information-outline`

Reply:
Because if people copy the config it will not work for them.

Reply:
So switch these to hass:? Or just revert them?

Reply:
Yeah, switch to `hass:`

Reply:
Will do when I get back to a PC

Reply:
Downside of switching this to `hass` is that we will now ship icons in our icon set to normal users that are only there because of the demo.

Reply:
They are already in the icon set, for now that is...

Reply:
Switched